setup.cfg

```diff
@@ -3,3 +3,8 @@ universal = 1
 
 [metadata]
 license_file = LICENSE
+
+[options]
+setup_requires =
```

Comment:
You know that with old setuptools this won't work, right?

Reply:
Yup. But deployments happen with Python 3.7 and modern setuptools. We're in the slow process of starting the migration of getting ready to drop Python 2.7, but our field (high energy physics) is still using older everything so we haven't gone Python 3 and modern libs only yet.

Reply:
Well, as long as you ship binary wheels building them yourself it shouldn't matter.